src/components/Comments.jsx

```diff
@@ -1,14 +1,38 @@
-// import { useEffect, useState } from "react";
-// import { fetchCommentsByArticleId } from "../api";
+import { useEffect, useState } from "react";
+import { fetchCommentsByArticleId } from "../api";
 
-// export default function Comments() {
-//   const [comments, setcomments] = useState([]);
+export default function Comments() {
+  const [comments, setcomments] = useState([]);
 
-//   useEffect(() => {
-//     fetchCommentsByArticleId(1).then((res) => {
-//       setcomments(res.comments);
-//     });
-//   });
-// }
+  useEffect(() => {
+    fetchCommentsByArticleId(1).then((res) => {
```

Comment:
This is hard coded to article one. All comments on all articles will be the same. This needs to be parametric to the article the user is viewing